.travis.yml

```diff
@@ -0,0 +1,256 @@
+language: python
+python:
+  - "3.8.2"
+
+os: linux
+dist: xenial
+
+services:
+  - docker
+
+import:
+  - Alfresco/alfresco-process-tools:.travis.helm.yml@master
+  - Alfresco/alfresco-process-tools:.travis.kubernetes_install.yml@master
+  - Alfresco/alfresco-process-tools:.travis.docker_login.yml@master
+
+env:
+  global:
+    - TRAVIS_WAIT_TIMEOUT=${TRAVIS_WAIT_TIMEOUT:-30}
+    - BRANCH=${TRAVIS_PULL_REQUEST_BRANCH:-${TRAVIS_BRANCH}}
+    - HELM_REPO_BASE_URL=https://kubernetes-charts.alfresco.com
+    - HELM_REPO=incubator
+    - KUBERNETES_VERSION=1.18.4
+    - HELM_VERSION=${HELM_VERSION:-3.2.4}
+
+branches:
+  only:
+    - master
+    - OPSEXP-115
+    - /support.*/
```

Comment:
Do we want this for support and feature branches?

Reply:
Ok I guess not for support  but why not for feature branches?

Reply:
Sorry, I'm getting myself confused with hotfix branches, you're right, feature branches are fine.